src/SerilogCommonLogger.cs

```diff
@@ -562,14 +562,15 @@ protected void Write(LogLevel level, Exception exception, object message)
         {
             var messageHandler = new FormatMessageHandler(delegate(string message, object[] parameters)
             {
-                string formatted = string.Format(formatProvider, message, parameters);
-
                 if (formatProvider == null)
                     WriteFormat(level, exception, message, parameters);
                 else
+                {
+                    string formatted = string.Format(formatProvider, message, parameters);
                     Write(level, exception, formatted);
+                }
 
-                return formatted;
+                return string.Empty; // Not used
```

Comment:
Since the Write/WriteFormat methods are called directly from the FormatMessageHandler, we never actually use the returned string. So no need to return it.